### ISSUE TITLE
请问Cpp部署可以实现TIDL加速吗 

### ISSUE THREAD

**Issue by linzhuoyingxmu** [TexasInstruments/edgeai-tidl-tools#79]

**Comment by vtrip97** [comment]
@linzhuoyingxmu : Yes - CPP deployment can achieve acceleration on the C7X DSP as long as all the nodes are offloaded to the DSP. I recommend going through [python examples](https://github.com/TexasInstruments/edgeai-tidl-tools/tree/master/examples/osrt_python) for model compilation (You can see what nodes get offloaded vs what run on ARM for your model) & [CPP Examples](https://github.com/TexasInstruments/edgeai-tidl-tools/tree/master/examples/osrt_cpp) for deployment
